src/simulator/scenarios/Scenarios.ts

```diff
@@ -1,15 +1,19 @@
 import { InGameEvent } from '../in-game-events/InGameEvents';
 import { CapabilityImprovements, ContainmentPolicy } from '../player-actions/PlayerActions';
-import { PlayerActions } from '../SimulatorState';
+import { Indicators } from '../SimulatorState';
 import { VictoryCondition } from '../victory-conditions/VictoryConditon';
 /**
  * Represents the initial state of the game world.
  */
 export interface Scenario {
     totalPopulation: number;
     initialNumInfected: number;
+    initialMedicalCosts: number;
+    initialEconomicCosts: number;
```

Comment:
These values should correspond to the values in the last item of the `runUpPeriod`